homeassistant/components/weather/__init__.py

```diff
@@ -81,6 +82,8 @@
 ATTR_FORECAST_WIND_BEARING: Final = "wind_bearing"
 ATTR_FORECAST_NATIVE_WIND_SPEED: Final = "native_wind_speed"
 ATTR_FORECAST_WIND_SPEED: Final = "wind_speed"
+ATTR_FORECAST_NATIVE_DEWPOINT: Final = "native_dewpoint"
```

Comment:
Isn't it "dew point" and not "dewpoint"?

<https://en.wikipedia.org/wiki/Dew_point>

Reply:
You're probably right, I see both when I "google" but it appears the links from wikipedia seems mostly as "Dew point" so I will change this.

Reply:
![CleanShot 2023-06-22 at 19 38 57](https://github.com/home-assistant/core/assets/195327/80e373e7-8aaf-43cc-ab3b-8764c04883fe)

![CleanShot 2023-06-22 at 19 38 36](https://github.com/home-assistant/core/assets/195327/49b3cddb-1891-4dd0-8694-f7f99a056c41)

I'm not 100% sure either, however, the space-separated one does have more hits

Reply:
Also google translate does my swedish word to "Dew point" so I would think it's inline what most use.
Has now changed